packages/jest-cli/src/runJest.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+import type {PatternInfo} from './SearchSource';
+import type {HasteContext} from 'types/HasteMap';
+
+const realFs = require('fs');
+const fs = require('graceful-fs');
+fs.gracefulify(realFs);
```

Comment:
I don't think we need this here, it already happens in `jest.js` which should be the entry point for Jest.